sound/soc/sof/ops.h

```diff
@@ -470,6 +470,14 @@ snd_sof_set_mach_params(const struct snd_soc_acpi_mach *mach,
 		sof_ops(sdev)->set_mach_params(mach, dev);
 }
 
+/* client register */
```

Comment:
@ranj063 Leave the comment out, doesn't seem to add anything to the code itself.